util/msys2_install.sh

```diff
@@ -8,25 +8,27 @@ installflip=false
 util_dir=$(dirname "$0")
 
 echo "Installing dependencies needed for the installation (quazip)"
-pacman --needed -S base-devel mingw-w64-x86_64-toolchain mingw-w64-x86_64-clang msys/git msys/p7zip msys/python3 msys/unzip
+pacman --needed --noconfirm --disable-download-timeout -Sy base-devel mingw-w64-x86_64-toolchain mingw-w64-x86_64-clang msys/git msys/p7zip msys/python3 msys/unzip
 
 source "$dir/win_shared_install.sh"
 
 function install_avr {
     rm -f -r "$avrtools"
-    wget "http://ww1.microchip.com/downloads/en/DeviceDoc/avr8-gnu-toolchain-3.6.1.1752-win32.any.x86.zip"
+    wget "https://blog.zakkemble.net/download/avr-gcc-8.3.0-x86-mingw.zip"
     echo "Extracting AVR toolchain..."
-	unzip -q avr8-gnu-toolchain-3.6.1.1752-win32.any.x86.zip
-	mv avr8-gnu-toolchain-win32_x86/ avr8-gnu-toolchain
-    rm __MACOSX -R
-    rm avr8-gnu-toolchain-3.6.1.1752-win32.any.x86.zip
-    pacman --needed -S mingw-w64-x86_64-avrdude
+    unzip -q -d . avr-gcc-8.3.0-x86-mingw.zip
+    mv avr-gcc-8.3.0-x86-mingw avr8-gnu-toolchain
+    rm avr8-gnu-toolchain/bin/make.exe
+    rm avr-gcc-8.3.0-x86-mingw.zip
+    pacman --needed --disable-download-timeout -S mingw-w64-x86_64-avrdude
 }
 
 function install_arm {
-    wget -O gcc-arm-none-eabi.zip "https://developer.arm.com/-/media/Files/downloads/gnu-rm/6-2017q2/gcc-arm-none-eabi-6-2017-q2-update-win32.zip?product=GNU%20ARM%20Embedded%20Toolchain,ZIP,,Windows,6-2017-q2-update"
-    unzip -d gcc-arm-none-eabi gcc-arm-none-eabi.zip
-    rm gcc-arm-none-eabi.zip
+    rm -f -r "$armtools"
+    wget -O gcc-arm-none-eabi-8-2019-q3-update-win32.zip "https://developer.arm.com/-/media/Files/downloads/gnu-rm/8-2019q3/RC1.1/gcc-arm-none-eabi-8-2019-q3-update-win32.zip?revision=2f0fd855-d015-423c-9c76-c953ae7e730b?product=GNU%20Arm%20Embedded%20Toolchain,ZIP,,Windows,8-2019-q3-update"
```

Comment:
url can be reduced slightly
```suggestion
    wget -O gcc-arm-none-eabi-8-2019-q3-update-win32.zip "https://developer.arm.com/-/media/Files/downloads/gnu-rm/8-2019q3/RC1.1/gcc-arm-none-eabi-8-2019-q3-update-win32.zip"
```